jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -40,7 +39,7 @@ module java.base {
   uses com.ibm.gpu.spi.GPUAssist.Provider;
   exports com.ibm.gpu.spi to openj9.gpu;
   // TODO remove the following after they move to their new modules
-  exports com.ibm.cuda;
-  exports com.ibm.gpu;
+  exports com.ibm.cuda to openj9.cuda;
```

Comment:
These packages should be moved to their respective modules. Adding these qualified exports is going to hide these packages that should be exported unconditionally. I suggest dealing with cuda and gpu is better left to another PR.

Reply:
i.e. #189

Reply:
okay.